ci-operator/templates/openshift/installer/cluster-launch-installer-custom-test-image.yaml

```diff
@@ -632,6 +653,41 @@ objects:
         TF_LOG=debug openshift-install --dir=/tmp/artifacts/installer create cluster 2>&1 | grep --line-buffered -v password &
         wait "$!"
 
+        if [[ "${CLUSTER_TYPE}" == "gcp" ]] && [[ "${CLUSTER_ENABLE_NESTED_VIRT-false}" == "true" ]]; then
+          ### enable nested-virt ###
+          export KUBECONFIG=/tmp/artifacts/installer/auth/kubeconfig
+          oc get machineset -n openshift-machine-api -o json >/tmp/machinesets.json
+          MACHINE_IMAGE=$(jq -r .items[0].spec.template.spec.providerSpec.value.disks[0].image /tmp/machinesets.json)
+          sed -i 's/'"$MACHINE_IMAGE"'/'"$CLUSTER_NESTED_VIRT_IMAGE"'/g' /tmp/machinesets.json
+          oc apply -f /tmp/machinesets.json
+          oc scale --replicas=0 machineset --all -n openshift-machine-api
+          oc get machines -n openshift-machine-api -o json >/tmp/machines.json
+          num_machines=$(jq '.items | length' /tmp/machines.json)
+          while [ "$num_machines" -ne "3" ]; do
+            sleep 5
+            oc get machines -n openshift-machine-api -o json >/tmp/machines.json
+            num_machines=$(jq '.items | length' /tmp/machines.json)
+          done
+          oc scale --replicas=1 machineset --all -n openshift-machine-api
+          while [ "$num_machines" -ne "6" ]; do
+            sleep 5
+            oc get machines -n openshift-machine-api -o json >/tmp/machines.json
+            num_machines=$(jq '.items | length' /tmp/machines.json)
+          done
+          while [ $(oc get nodes | wc -l) -ne "7" ]; do oc get nodes; sleep 5; done
+          nodes_ready=false
+          while ! "$nodes_ready"; do sleep 5 && if ! oc get nodes | grep NotReady; then nodes_ready=true; fi; done
+          oc project default
+
+          # apply kvm device plugin
+          oc apply -f https://raw.githubusercontent.com/kubevirt/kubernetes-device-plugins/master/manifests/kvm-ds.yml
+          workers=$(oc get nodes | grep worker | awk '{ print $1 }')
+          workers_each=($workers)
+          for i in {0..2}; do
+            if ! oc debug node/"${workers_each[i]}" -- ls /dev/kvm; then oc debug node/"${workers_each[i]}" -- ls /dev/kvm; fi
```

Comment:
_this no longer works_ `oc debug node ...` behavior has recently changed, so with a failed cmd passed, like so:
`oc debug node/nodename -- ls /dev/kvm` when nested-virt is not enabled, the cmd now exits non-zero (as it should always have).  This needs a rework, I have an ugly rework but feel free to make it better :)  
```
               workers=$(oc get nodes | grep worker | awk '{ print $1 }')
               workers_each=($workers)
               for i in {0..2}; do
                   for j in {0..3}; do
                       out=$(oc debug node/"${workers_each[i]}" -- ls /dev/kvm || true)
                       if [[ ! "${out}" =~ "No such file" ]]; then
                           break
                       else
                           if [[ $j -eq 3 ]]; then
                               echo "nested virt not enabled, exiting" 2>&1
                               exit 1
                           fi
                       fi
                   done
               done
```